src/client/Microsoft.Identity.Client/ApiConfig/AcquireTokenInteractiveParameterBuilder.cs

```diff
@@ -295,7 +295,7 @@ public AcquireTokenInteractiveParameterBuilder WithParentActivityOrWindow(IWin32
         }
 #endif
 
-#if DESKTOP || NET5_WIN || NET_CORE
+#if DESKTOP || NET5_WIN || NET_CORE || NETSTANDARD
```

Comment:
Why are these changes needed?

Reply:
Broker target is NetStandard and these are consumed by broker.

Reply:
Where at? I didn't find any usages. Guess just trying to figure out the order in which targets are chosen. I thought main Msal target would be determined by my app's target not Msal.Broker.

Also for some reason Msal.Desktop doesn't build for me (VS 19 and 22) when building the Net Core WAM dev app...
![image](https://user-images.githubusercontent.com/34331512/172782088-e8514fd7-116e-45e2-9ffe-ea7a0d5eff2b.png)

Reply:
taking a look at it, I am not able to repro it still. DevApps seems to building for me.